packaging/packager

```diff
@@ -56,26 +55,41 @@ if ! type zip > /dev/null 2>&1; then
     echo "zip utility is not found. Please install it and re-run this script"
     exit 1
 fi
-function package_libc_via_pacman {
+
+function pluck_so_files() {
+    sed -E '/\.so$|\.so\.[0-9]+$/!d'
+}
+
+function package_libc_alpine() {
+    if grep -F "Alpine Linux" < /etc/os-release > /dev/null; then
```

Comment:
Use the long form of the command line option, please.
`grep --fixed-strings`

Reply:
@marcomagdy, but now it doesn't build on Alpine, because busybox knows `grep -F` but it doesn't know it as `--fixed-strings`.